wpi_jaco:
  # Name of the arm. Will appear in topic names and services.
  arm_name: "jaco2"

  # Difference between values of the Mico and Jaco fingers, Jaco normalized to 1
  finger_scale: 1.0

  # Threshold in the JACO API's finger position units to consider a finger position reached
  finger_error_threshold: 1.0

  # Value of an open gripper
  gripper_open: 0.0

  # Value of a closed gripper
  gripper_closed: 65.0

  # For calculating splines in the joint trajectory
  max_curvature: 20.0

  # Maximal speed of the fingers while opening/closing
  max_speed_finger: 30.0

  # The number of fingers
  num_fingers: 3
